docs/perf_client.rst

```diff
@@ -216,6 +217,136 @@ tell perf\_client to send batch-size 4 requests of shape [ 3, 224, 224 ]::
 
   $ perf_client -m mymodel -b 4 --shape IMAGE:3,224,224
 
+Real Input Data
+^^^^^^^^^^^^^^^
+
+The performance of some of the models is highly dependent on the data used.
+For such cases users can provide data to be used with every inference request
+made by client in a json file. The client will keep cycling through the data
+to maintain steady inferences.
+
+Every data object should specify data for all the inputs with exact size
```

Comment:
"count" indicates the number of entries in the "data" array. Each entry in the "data" array must specify all input tensors with the exact size expected by the model."

Confusing to use string and then have the string be a number. Just use a INT32 datatype. Also use a 2-d shape and note that the data is flattened in row-major order.

Delete "Otherwise, ... thown".